vsutil/types.py

```diff
@@ -12,20 +12,26 @@
 
 class Dither(str, Enum):
     """
-    enum for zimg_dither_type_e
+    Enumeration for `zimg_dither_type_e`.
```

Comment:
I get that “Enumeration” is probably the correct term, but I’ve never met anyone who doesn’t just say enum, so this might cause more confusion than it prevents.